internal/app/maintenance/server/server.go

```diff
@@ -61,3 +64,27 @@ func (s *Server) ApplyConfiguration(ctx context.Context, in *machine.ApplyConfig
 
 	return reply, nil
 }
+
+// Disks implements machine.MaintenanceService.
+func (s *Server) Disks(ctx context.Context, in *empty.Empty) (reply *storage.DisksResponse, err error) {
```

Comment:
we need to add a registration line to the `Register` method

Reply:
ah, right :+1: